src/python/pants/init/plugin_resolver.py

```diff
@@ -51,9 +51,7 @@ async def resolve_plugins(
     `named_caches` directory), but consequently needs to disable the process cache: see the
```

Comment:
FYI changes to this file can be reverted. `apply_requirement_constraints` defaults to `False`.

Reply:
I'm going to leave Stu's comment and explicit False in in the follow-up since it looks like it marks danger in a useful way.